Common/Python/BrokerageModelPythonWrapper.cs

```diff
@@ -262,8 +283,7 @@ public ISettlementModel GetSettlementModel(Security security, AccountType accoun
         {
             using (Py.GIL())
             {
-                return (_model.GetSettlementModel(security, accountType)
-                    as PyObject).GetAndDispose<ISettlementModel>();
+                return (_model.GetSettlementModel(security, accountType) as PyObject).GetAndDispose<ISettlementModel>();
```

Comment:
Does this need the same `TryConvert`... like the other methods?

Reply:
Yeah, but since that method has been deprecated I didn't know if I should also modify it 🤔